skimage/util/tests/test_process.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+from numpy.testing import assert_array_almost_equal
+
+from skimage.filters import threshold_adaptive, gaussian_filter
+from skimage.util import process_chunks
+
+
+def test_process_chunks():
+    # data
+    a = np.arange(144).reshape(12, 12).astype(float)
+
+    # wrapp the function we're applying
+    def wrapped_thresh(arr):
+        return threshold_adaptive(arr, 3, mode='reflect')
```

Comment:
I typically use `functools.partial` for these cases.  We could also include `args=` and `kwargs=` keyword arguments to `process`.